src/parser/ast.c

```diff
@@ -145,7 +145,7 @@ AST_Validation _Validate_WHERE_Clause(const AST_Query *ast, char **reason) {
 
 AST_Validation AST_Validate(const AST_Query *ast, char **reason) {
   /* AST must include either a MATCH or CREATE clause. */
-  if (!(ast->matchNode || ast->createNode || ast->mergeNode)) {
+  if (!(ast->matchNode || ast->createNode || ast->mergeNode || ast->returnNode)) {
```

Comment:
This needn't be a huge priority right now, but we need to make a few further changes to support return clauses without match clauses. `_Validate_RETURN_Clause` requires one, and even with that removed, the proposed test from this suite - "RETURN ABS(-1)" -  does not yet work.

Reply:
Agree